src/questions.js

```diff
@@ -0,0 +1,160 @@
+const choiceQuestions = [
+  {
+    type: "list",
+    message: "What would you like to do?",
+    name: "choiceOption",
+    choices: [
+      {
+        value: "getEmployees",
+        name: "View All Employees",
+      },
+      {
+        value: "addEmployees",
+        name: "Add New Employee",
+      },
+      {
+        value: "updateRole",
+        name: "Update Employee Role",
+      },
+      {
+        value: "getRoles",
+        name: "View All Roles",
+      },
+      {
+        value: "addRole",
+        name: "Add New Role",
+      },
+      {
+        value: "getDepartment",
+        name: "View All Departments",
+      },
+      {
+        value: "addDepartment",
+        name: "Add New Department",
+      },
+      {
+        value: "quit",
+        name: "Quit Application",
+      },
+    ],
+  },
+];
+
+const getDepartmentQuestion = () => {
```

Comment:
why does this need to be wrapped in a function? We can just export the question directly from the file instead